RESTful_API.py

```diff
@@ -0,0 +1,60 @@
+"""Here we start with the construction of our FASTapi server. This server
+will be designed to add items to, delete items from, and retrieve a 
+given list of items. """
+
+from fastapi import FastAPI  
+from pydantic import (
+    BaseModel,
+)  
+
+""" Turns FastAPI into a function that we can pull commands from."""
+app = (
+    FastAPI()
+)  
+
+
+""" Make an item class for the API to work with."""
+class Item(BaseModel):  
+    name: str  
+    item_characteristics: str  
+
+
+""" Make a list of items to work with."""
+my_list = [
+    ["name", "components"]
+]  
+
+
+"""Now we have a list to work with, let's make our necessary endpoints that work with it."""
+
+""" Adds a new item to the list.
+
+This function allows the user to specify details of an item
+they would want added to the list, adds those items to the 
+list, and then returns the list's most recent addition. """
+@app.post("/new-item")
+def new_item(item: Item):
+    my_list.append(item)  
+    return my_list[-1]  
```

Comment:
Docstrings are special. They are "comments" in the sense that they are not something that gets executed but they are also visible to the python interpreter (comments are invisible). This allows us to do cool things like generate documentation for methods based on the docstring and have python extract those docstrings for use (no manually parsing the python file with another tool). BUT. This means we have to follow the rules for docstrings in order to gain all these benefits.

Python docstrings are placed _inside_ of the documented method or class:
```suggestion
@app.post("/new-item")
def new_item(item: Item):
    """ Adds a new item to the list.

    This function allows the user to specify details of an item
    they would want added to the list, adds those items to the 
    list, and then returns the list's most recent addition.
    """
    my_list.append(item)  
    return my_list[-1]  
```